core/src/block.rs

```diff
@@ -577,7 +577,6 @@ impl ValidBlock {
     ///
     /// # Panics
     /// If generating keys or block signing fails.
-    #[cfg(test)]
```

Comment:
Is is now used outside of tests? Or is this related to the problem that Marin was having, with `#[cfg(test)]` not working properly on some toolchains?

Reply:
AFAICT without deleting this line, I could not import this method to the tests

Reply:
It's the same bug as with @mversic

Reply:
this is getting out of hands

Reply:
ok, this is not related to my bug as far as I can tell.  This is not a bug at all.
This is happening because `cfg(test)` is true only for the crate you are testing, not the crates which are dependencies. This can be resolved either like this and have `new_dummy` exposed in the api or you can make a cargo feature in the `iroha_core` which is activated only during testing

Reply:
I would rather copy that code here if possible and leave `iroha_core` as it is. `new_dummy` should not be used from outside `iroha_core` crate